recipes/bzip2/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: bzip2
+  version: 1.0.6
+
+source:
+  fn: bzip2-1.0.6.tar.gz
+  url: http://www.bzip.org/1.0.6/bzip2-1.0.6.tar.gz
+  md5: 00b516f4704d4a7cb50a1d97e6e8e15b
```

Comment:
Please change to another checksum. Preferably to `sha256` though anything they give us that is not `md5` could work too.